opennlp-tools/src/main/java/opennlp/tools/formats/irishsentencebank/IrishSentenceBankDocument.java

```diff
@@ -45,7 +45,7 @@
  * of tokenized sentences.
  * <p>
  * The sentence bank can be downloaded from, and is described
- * <a href="http://www.lexiconista.com/datasets/sentencebank-ga/">here</a>
+ * <a href="http://www.lexiconista.com/datasets/sentencebank-ga/">here</a>.
```

Comment:
Deadlink ? I get a GitHub 404 for it

Reply:
Alternatives?

Reply:
I think it is this one https://github.com/michmech/irish-sentence-bank (from the same person)

Reply:
Resolved with next commit.